docs/core-concepts/stability-and-exploratory-testing.mdx

```diff
@@ -4,6 +4,106 @@ sidebar_label: 'Stability and Exploratory Testing'
 
 # Stability and Exploratory Testing
 
-import Placeholder from '../partials/_placeholder.mdx';
+Regression Games permits more than just predefined testing scenarios. Stability and exploratory testing uses bots can be used to find bugs when you may not have a specific test case in mind.
 
-<Placeholder />
\ No newline at end of file
+## Monkey Bots
+
+<div style={{position: "relative", paddingBottom: "50.40485829959515%", height: 0}}><iframe src="https://www.loom.com/embed/8cde5bbe454c497aa74bd3afacca23cc?sid=745912d2-c555-4ef3-8a24-b2befd49c808" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: "absolute", top: 0, left: 0, width: "100%", height: "100%"}}></iframe></div>
```

Comment:
I like this video! Flows well and it demonstrates the approach. However, we can consider adding a little bit more of a flare.

I think it would have been cool if we could intentionally add a bug (maybe a simple null reference exception) and end the video with having the monkey bot finding it. Another extra step might even include going to the website dashboard and using the monkey bot recording to point out an interesting <thing> whatever that might be.

I really appreciate tutorials that, in addition to showing how to use a tool, it also teaches me how to get value out of a tool.